### PR TITLE
Renamed all classes, functions and variables to adoption

### PR DESCRIPTION
**Pull request by florisdobber**
A separate PR will be put up to adjust the documentation 
Ticket: https://code4community-team.monday.com/boards/925293934/pulses/1218648886

EDIT: Documentation PR https://github.com/Code-4-Community/c4c-dev-docs/pull/56

### CODE REVIEW

**Review by liammoyn — APPROVED** [review]
Curious why we're getting rid of duration?

**Comment by florisdobber** [comment]
> 
> 
> Curious why we're getting rid of duration?

Speak for the Trees doesn't want it anymore

File: api/src/main/java/com/codeforcommunity/api/ISiteProcessor.java [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 import com.codeforcommunity.auth.JWTData;
-import com.codeforcommunity.dto.site.FavoriteSitesResponse;
+import com.codeforcommunity.dto.site.AdoptedSitesResponse;
 import com.codeforcommunity.dto.site.RecordStewardshipRequest;
 import com.codeforcommunity.dto.site.StewardshipActivitiesResponse;
 
 public interface ISiteProcessor {
 
   /** Creates a record in the favorite sites table linking the user and the site */
```
Comment (SofieDunt) [inline api/src/main/java/com/codeforcommunity/api/ISiteProcessor.java:10]: Should rename to adopted sites table in comments

**Review by SofieDunt — APPROVED** [review]
Thanks for renaming everything!

File: api/src/main/java/com/codeforcommunity/api/ISiteProcessor.java (hunk 1 above)
Comment (florisdobber) [reply api/src/main/java/com/codeforcommunity/api/ISiteProcessor.java:10]: Good catch, fixed

**Pull request merged**